.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: python
+python: "2.7"
+before_install:
+ - sudo apt-get update -qq
+ - sudo apt-get install -qq python-apt python-pycurl
+install:
+  - pip install ansible==1.8.0
```

Comment:
Just for information. 
This is minimal version that role can run on travis?
If you don't tested lower version - I'll do this after merge and sync with `meta/main.yml` (by the way I'm not sure my role work with 1.4 as I have only 1.9 on my computer).